wiki/Options/en.md

```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
 | Name | Description |
 | --- | --- |
 | `VSync` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
-| `120fps` | Limits the game client to 120fps. |
-| `240fps` | Limits the game client to 240fps. |
+| `Power Saving` | Limits the game client to twice the refresh rate of your monitor. |
+| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |
 | `Unlimited (gameplay)` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
 
 - `VSync` explanation: In layman terms, using VSync will force the game wait for the entire frame to load before displaying it.
   - You may want to use this if you see some "tearing" (when the bottom portion of the game is lagging behind the upper portion of the game).
   - Despite what the point above stated, this may cause some lag or slowdowns because the game client has to wait for each frame to load.
 - `Unlimited (gameplay)` explanation: Unlimited only applies to when you are playing a beatmap.
-  - When you are doing anything but playing a beatmap, `240fps` is used instead.
+  - When you are doing anything but playing a beatmap, the frame rate is limited to 240fps.
+  - This option is **not recommended**. Using unlimited frame limiter can lead to stutters.
+  - By using the `Optimal` setting, you will be automatically set to a frame limiter which (if your hardware can reach) offers imperceivable changes in system (input to output) latency when compared to `unlimited`.
```

Comment:
```suggestion
  - Using the `Optimal` option instead offers imperceivable changes in system (input to output) latency when compared to `Unlimited (gameplay)`.
```

I think we can shorten this up to what I suggested. The `Optimal` setting was explained in the table above.